### DISCUSSION TITLE
Inconsistent character set supported by the default font across platforms

### DISCUSSION THREAD

**Discussion by lvella** [hecrj/iced#1097] [Q&A] [answered]
I have developed an application in Linux, and used some smiley emoji characters in the text. When I built it in Windows and macOS, the emoji rendered as a rectangle. I am not sure what can be done on Iced side about this, but having an inconsistent character set supported by the default font across platform harms its portability.

**Comment by hecrj** [comment, marked as answer]
`iced` will try to load a system font by default. Thus, until we implement proper font fallback (see #33), the supported character set depends on the system.

If you want a reliable set of supported characters, then I recommend you to embed your own font and provide it to [`Settings::default_font`](https://docs.rs/iced/0.3.0/iced/settings/struct.Settings.html#structfield.default_font).
